crates/core_arch/src/x86/sse.rs

```diff
@@ -1371,6 +1371,10 @@ pub unsafe fn _mm_sfence() {
 /// For more info see [`_mm_setcsr`](fn._mm_setcsr.html)
 ///
 /// [Intel's documentation](https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_getcsr)
+#[rustc_deprecated(
+    since = "1.40.0",
```

Comment:
```suggestion
    since = "1.59.0",
```
this is the current nightly.

Reply:
Whoops, I didn't catch that! Thanks.